.gitlab-ci.yml

```diff
@@ -1 +1,4 @@
-include: 'https://projects.govcms.gov.au/govcms/govcms8-scaffold-ci/raw/master/.gitlab-ci-include.yml'
+include:
+  - project: 'govcms/govcms8-scaffold-ci'
+    ref: canary_test
```

Comment:
This is making me pause. Is the purpose of the different branch to just having a non-prod version we can fork? If that's the case, maybe we can leave it on master, and just add a variation file called `.gitlab-ci-include-canary.yml`. This way when we are working on the files we are really clear about making sure certain changes are reflected in both files. Not sure.

Reply:
It kinda leaves us in a state where someone needs to know that `canary_test` is being referenced elsewhere, it doesn't look like the processor will fallback to `HEAD` if ref doesn't exist. https://gitlab.com/gitlab-org/gitlab-ce/blob/master/lib/gitlab/ci/config/external/processor.rb

If it gets deleted this will potentially break.

Reply:
If I remove this change completely does it prevent canary testing? I don't think so. This is 4 weeks ago so I'm a bit hazy but it definitely looks like it should default to master.

Reply:
We should continue to use master here -- the `canary_test` branch on gitlab is stale; and actually I'm not even sure _why_ we wanted to use a different branch in the first place (canary builds should use the same branches as any other builds)

If you can revert the `.gitlab-ci.yml` file to current master then we can merge this.

Reply:
Done